### DISCUSSION TITLE
Iframe integration: Disable tracking up until first interaction

### DISCUSSION THREAD

**Discussion by mquandalle** [plausible/analytics#1648] [Anything Else]
Hi,

One of my website can be integrated with an iframe on third-parties applications. Oftentimes the iframe is at the end of the page, and users don't interact with it. However plausible still count them as “visitors” of my site.

For the iframe integration I would like to disable plausible tracking on page load up until the user interact with the site (example: click somewhere).

I suppose I could do that by removing the plausible script from the page on load, and inserting it with Javascript when desired, but maybe there is a better implementation of the same behavior using plausible client API?

**Comment by metmarkosaric** [comment]
hi @mquandalle! we do the have the manual script extension that could be useful for this use case. see https://plausible.io/docs/script-extensions#plausiblemanualjs

**Reply by mquandalle** [reply]
Thank you for the quick answer! Is there still a way to re-enable automatic “history-tracking” in manual mode? I need it after the first interaction or when the site isn't loaded in an iframe.

**Reply by ukutaht** [reply]
Hey @mquandalle! Unfortunately there's no way to do this cleanly at the moment, sorry!

Could you try the method you mentioned previously?

> I suppose I could do that by removing the plausible script from the page on load, and inserting it with Javascript when desired, but maybe there is a better implementation of the same behavior using plausible client API?

**Reply by mquandalle** [reply]
Yes, this is what I ended up doing and it works just fine https://github.com/mquandalle/mesaidesvelo/commit/8f279ada8038ef3b2fa3a26b4e0539c3fa3c1aba
